cql/src/main/java/io/stargate/cql/CqlActivator.java

```diff
@@ -54,8 +71,12 @@ protected ServiceAndProperties createService() {
     if (cql != null) { // Already started
       return null;
     }
+
+    healthCheckRegistry.get().register(HEALTH_CHECK_NAME, healthCheck);
+
     cql = new CqlImpl(makeConfig(), persistence.get(), metrics.get(), authentication.get());
     cql.start();
+    started.set(true);
```

Comment:
Do we want to define "started" vs "running"? Like after calling `start()` it'll take a little bit before it's ready to start handling connections. 

_You might have this and I just haven't gotten to it yet_

Reply:
I'll double check. At first glance it looks like the TCP stuff should be ready by the time `cql.start()` returns :thinking:

Reply:
`CqlImpl` calls `persistence.setRpcReady(true)` before returning from `start()` I assume this means we're ready for client requests at this point.

Reply:
CQL was just an example. Probably a bigger issue for rest and graphql since they take a little longer to start up.

Reply:
GraphQL preloads its cache before returning from "start" (verified in debugger). As for REST it appears to be pretty stateless.

We will certainly have some period of slow operation before JIT kicks in, but warming it up is probably too much for this PR's scope WDYT?